master/setup.py

```diff
@@ -391,6 +391,10 @@ def define_plugin_entries(groups):
             'pylint==1.1.0',
             'pyflakes',
         ],
+        'bundle': [
+            'buildbot-www',
```

Comment:
it does not really as far as pip is involved. this is only a build dependancy, so it is not in the deps.
I verified buildbot-www.whl depends on nothing.